src/components/application_manager/test/commands/hmi/hmi_notifications/hmi_notifications_test.cc

```diff
@@ -239,6 +239,13 @@ class HMICommandsNotificationsTest
   void SetUp() OVERRIDE {
     message_helper_mock_ =
         application_manager::MockMessageHelper::message_helper_mock();
+    Mock::VerifyAndClearExpectations(message_helper_mock_);
+  }
+
+  void TearDown() OVERRIDE {
+    // Fix DataAccessor release and WinQt crash
+    Mock::VerifyAndClearExpectations(&mock_app_manager_);
+    Mock::VerifyAndClearExpectations(message_helper_mock_);
```

Comment:
@vlantonov Why are these two calls in the `TearDown()` here and in destructors in other files? Isn't it possible that they are put in the destructor here as well?